packages/components/src/dropdown-menu-v2/index.tsx

```diff
@@ -75,7 +75,7 @@ const UnconnectedDropdownMenu = ( props: DropdownMenuProps ) => {
 
 	// Render the portal in the default slot used by the legacy Popover component.
 	const slot = useSlot( POPOVER_DEFAULT_SLOT_NAME );
-	const portalContainer = slot.ref?.current;
+	const portalContainer = slot.ref?.current || null;
```

Comment:
https://github.com/WordPress/gutenberg/blob/3b4ae6043f422cfbd60abd0e2aaac1cd227422e2/packages/components/src/dropdown-menu-v2/index.tsx#L47-L51

portalContainer is not undefined.